giskard/datasets/metadata/text_metadata_provider.py

```diff
@@ -35,7 +35,7 @@ def _detect_charset(text: str):
     return charset or "undefined"
 
 
-def _detect_lang(text: str):
+def detect_lang(text: str):
```

Comment:
maybe worth moving this out, in a shared module

Reply:
I move it into `giskard/utils/language_detection.py`